app/models/question.rb

```diff
@@ -34,6 +34,10 @@ def self.with_basic_association
     by_date.includes(user: [:social_providers])
   end
 
+  def self.from_user_subscription(user_tags)
+    joins(:tags).where(tags: { name: user_tags })
```

Comment:
I think we may need to look into custom sql to fetch this, also I think we may prefer order by the tags in case there are no questions in the tags a user is subscribed to(http://stackoverflow.com/questions/866465/order-by-the-in-value-list). Scratch it, front end would take care of that: (when there are no questions in the user subscription, they should come to top questions). But we still need to use custom sql to avoid about 3 db calls here